task_1.py

```diff
@@ -0,0 +1,35 @@
+phone_book = {}
+
+while True:
+
+    name = input("Enter name:")
+    if name == "End":
+        break
+    second_name = input("Enter second name:")
+    phone_number = input("Enter phone number:")
+    birthday = input("Enter date of birth:")
+
+    phone_book[name] = {"second name:": second_name, "phone number:": phone_number,
+                        "date of birth:": birthday}
+
+# add new customer
+
+add = phone_book["Vova"] = "+3801"
```

Comment:
there should be a program that reads user data from `input` like "add Vitalii 2342134" and parses that data and adds it to the phone_book. And other commands should work the same. For example
```
command, name, phone = users_data.split()
if command == "add":
    if name not in phone_book:
        phone_book[name] = phone
elif ...
```